sci-electronics/iverilog/iverilog-10.3.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+GITHUB_PV=$(ver_rs 1- '_')
+
+DESCRIPTION="A Verilog simulation and synthesis tool"
+HOMEPAGE="http://iverilog.icarus.com/"
+LICENSE="LGPL-2.1"
```

Comment:
One empty line here, let's try to follow the skel format.
```
EAPI=7

DESCRIPTION=""
HOMEPAGE=""
SRC_URI=""

LICENSE=""
SLOT="0"
KEYWORDS="~amd64 ~x86"
IUSE=""

DEPEND=""
RDEPEND="${DEPEND}"
BDEPEND=""
```

Reply:
> One empty line here, let's try to follow the skel format.

Thank you! I try my best to rearranged the lines to follow skel
https://github.com/gentoo/gentoo/pull/14096/files#diff-4a54e9b4d7b957bf5481638db78ae331R10-R24